Search/SearchPlugins/default/duckduckgo.xml

```diff
@@ -9,11 +9,11 @@
 <!-- this is effectively x-moz-phonesearch, but search service expects a text/html entry -->
 <Url type="text/html" method="GET" template="https://duckduckgo.com/">
   <Param name="q" value="{searchTerms}"/>
-  <Param name="t" value="ffocus"/>
+  <Param name="t" value="ffip"/>
```

Comment:
I think this should be `ffit` as well

Reply:
Just to add to it, as I see that it's very confusing in the Jira ticket: I confirmed with Kapil that we should use `ffit` for all of Focus/Klar on iOS, regardless of device type.

Reply:
Ok, will change then, TY!

Reply:
Just to confirm I agree with Martin, aiui the tablet/phone separations are being removed for DDG.

I'm wondering if we can therefore also remove the application/x-moz-tabletseearch section below (as they'll be the same).

Reply:
With my other comment about the overlay section/imports, it might be better to leave the application/x-moz-tabletsearch section for now.